felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -4061,6 +4145,15 @@ func (m *bpfEndpointManager) getIfaceTypeFromLink(link netlink.Link) IfaceType {
 	return IfaceTypeData
 }
 
+func (m *bpfEndpointManager) getBondSlaves(masterIfName string) []string {
+	slaveDevices := []string{}
```

Comment:
you could assume that if this is called only for bonds there are usually at least 2 slaves. Perhaps 4, you could preallocate capacity for that many and that would make the append a bit faster. But you may not need to allocate it at all and just iterate over the set.